src/modules/ai/aiVehicleSpline.h

```diff
@@ -12,16 +12,19 @@ namespace MM2
     // External declarations
     extern class AudImpact;
     extern class aiRailSet;
+    extern class aiVehicleInstance;
 
     // Class definitions
 
     class aiVehicleSpline : public aiVehicle {
     private:
         hook::Field<0x10, aiRailSet> _railSet;
+        hook::Field<0xD4, aiVehicleInstance*> _vehInstance;
         hook::Field<0xE6, short> _playerId;
         hook::Field<0xF4, float> _curSpeed;
+        hook::Field<0xFC, float> _wheelRotation;
     protected:
-        byte _buffer[0x184];
+        byte _buffer[0x180];
```

Comment:
Just confirming this is a bugfix and not because more fields were added?

Reply:
This one has changed because of the fields I added

Reply:
Yeah these ones